compass/landice/tests/ensemble_generator/plot_ensemble.py

```diff
@@ -202,61 +310,149 @@
 
     fpath = run + "/output/globalStats.nc"
     if os.path.exists(fpath):
+        n_with_output += 1
         f = netCDF4.Dataset(fpath, 'r')
         years = f.variables['daysSinceStart'][:] / 365.0
 
-        VAF = f.variables['volumeAboveFloatation'][:]
-        SLR = (VAF[0] - VAF) / 3.62e14 * rhoi / rhosw * 1000.
+        final_time[idx] = years[-1]
+
+        VAF = f.variables['volumeAboveFloatation'][:] / \
+            (1.0e12 / rhoi)  # in Gt
         totalArea = f.variables['totalIceArea'][:] / 1.0e6  # in km2
         fltArea = f.variables['floatingIceArea'][:] / 1.0e6  # in km2
         grdArea = f.variables['groundedIceArea'][:] / 1.0e6  # in km2
-        groundingLineFlux = f.variables['groundingLineFlux'][:] / 1.0e12  # Gt
-        BMB = f.variables['totalFloatingBasalMassBal'][:] / -1.0e12  # in Gt
-
-        # find target year index
-        indices = np.nonzero(years >= targetYear)[0]
-
-        # color lines depending on if they match obs or not
-        col = 'k'
-        alph = 0.2
-        GLobs = qoi_info['GL flux']['obs']
-        if GLobs is not None and len(indices) > 0:
+        iceArea = f.variables['totalIceArea'][:] / 1000.0**2  # in km^2
+        grdVol = f.variables['groundedIceVolume'][:] / (1.0e12 / rhoi)  # in Gt
+        BMB = f.variables['totalFloatingBasalMassBal'][:] / -1.0e12  # in Gt/yr
+        grSMB = f.variables['totalGroundedSfcMassBal'][:] / -1.0e12  # in Gt/yr
+        groundingLineFlux = f.variables['groundingLineFlux'][:] \
+            / 1.0e12  # Gt/yr
+        GLMigFlux = f.variables['groundingLineMigrationFlux'][:] \
+            / 1.0e12  # Gt/yr
+        w = 50
```

Comment:
Should this be `len(groundingLineFlux)` so that `groundingLineFlux` and GLMigFlux2` are the same size in line 334?

Reply:
w is the width of a boxcar filter applied to smooth this time series.  I've added a comment explaining that.